shared/ff_util/src/config_client/config_client.cc

```diff
@@ -21,8 +21,6 @@
 
 #include <vector>
 
-FF_DEFINE_LOGGER("config_client");
-
 namespace ff_util {
 
 ConfigClient::ConfigClient(NodeHandle & nh, const std::string &node) {
```

Comment:
Should this get moved to the .h file since the rest of the functions were already moved there.

Reply:
Brian suggested we use an Error() function such that we can define this here without redefining the variable :)